BookTracking.Web/src/pages/Home/BookTable.css

```diff
@@ -0,0 +1,13 @@
+﻿.row-expanded {
```

Comment:
CSS styles are imported globally in React. Ie another component rendered anywhere in the app would be able to use these classes. So its good practice to nest classes that dont need to be shared to scope them to your component. Something like `.book-table` and set the class name on the outermost returned element in `BookTable`